tests/providers/amazon/aws/sensors/test_redshift.py

```diff
@@ -56,6 +57,20 @@ def test_poke(self):
         )
         assert op.poke(None)
 
+    @unittest.skipIf(mock_redshift is None, 'mock_redshift package not present')
+    @mock_redshift
+    def test_poke_with_cluster_state(self):
+        self._create_cluster()
+        op = AwsRedshiftClusterSensor(
+            task_id='test_cluster_sensor',
+            poke_interval=1,
+            timeout=5,
+            aws_conn_id='aws_default',
+            cluster_identifier='test_cluster',
+            target_status=RedshiftClusterStates.AVAILABLE,
```

Comment:
so you already have a test that checks for `available`... i'm trying to think how you can verify the new behavior, namely that it will wait for whatever state you put here --- maybe you can think of a good way